packages/create-app/src/__snapshots__/expected-template-next-static-export/src/lib/client.ts

```diff
@@ -1,14 +1,30 @@
 import { FoundryClient, PublicClientAuth } from "@fake/sdk";
 
+const url = process.env.NEXT_PUBLIC_FOUNDRY_API_URL;
+const clientId = process.env.NEXT_PUBLIC_FOUNDRY_CLIENT_ID;
+const redirectUrl = process.env.NEXT_PUBLIC_FOUNDRY_REDIRECT_URL;
```

Comment:
Not doing a dynamic lookup based on the key as it may interfere with the env variable being bundled during build

https://nextjs.org/docs/pages/building-your-application/configuring/environment-variables#bundling-environment-variables-for-the-browser